src/whats-the-catch.md

```diff
@@ -0,0 +1,49 @@
+# OK, but what's the catch?
+
+---
+
+## You can't write C in Rust
+
+* You have to think about memory up-front
+  * Who owns any given value?
+  * Who needs to borrow it and when?
+  * Does it live long enough to satisfy those borrows?
+  * Are you borrowing something that might move?
+
+---
+
+## Rust exposes underlying complexity
```

Comment:
... and prioritizes managing it. 
Maybe a conclusion for the slide notes.

Reply:
How do I add slide notes?

Reply:
That is probably a question for @miguelraz